lms/djangoapps/courseware/views/views.py

```diff
@@ -939,6 +939,15 @@ def _get_cert_data(student, course, course_key, is_active, enrollment_mode):
 
     cert_downloadable_status = certs_api.certificate_downloadable_status(student, course_key)
 
+    if cert_downloadable_status['is_dateunavailable']:
```

Comment:
I think that this code here is why we don't want to return this status for any learners who haven't already passed the course on the verified track. It would be strange to see this message as an audit learner, for example.